content/en/docs/distributions/operator/install-kubeflow.md

```diff
@@ -4,7 +4,7 @@ description = "Instructions for Kubeflow deployment with Kubeflow Operator"
 weight = 10
 +++
 
-This guide describes how to use the Kubeflow Operator to deploy Kubeflow. As mentioned in the Operator [introduction](/docs/methods/operator/introduction.md), the Operator also allows you to monitor and manage the Kubeflow installation beyond the initial installation.
+This guide describes how to use the Kubeflow Operator to deploy Kubeflow. As mentioned in the Operator [introduction](/docs/distributions/operator/introduction.md/), the Operator also allows you to monitor and manage the Kubeflow installation beyond the initial installation.
```

Comment:
it works for me if removing the trailing slash

Reply:
Thanks, I will correct